store_page.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
The meta tag for description is missing again on this page as well. Otherwise, all else looks good.
1. Code is readable, as it's commented and spaced well
2. Code is functional